config/config.sample.yaml

```diff
@@ -126,6 +126,9 @@ provisioning:
 puppeting:
   # Should the bridge allow users to puppet their accounts
   enabled: false
+  # Should the bridge send users a onboarding message when
+  # they join a Slack channel for the first time.
+  onboard_users: false
```

Comment:
I think this should suggested to be on by default. 
```suggestion
  onboard_users: true
```

✨🐕‍🦺 First time user experience 💖🌟

Reply:
Okay, on for new bridges but off for existing?

Reply:
Yeah, lets default to off if this is undefined.